warehouse/upload.go

```diff
@@ -459,16 +461,19 @@ func (job *UploadJobT) loadAllTablesExcept(skipPrevLoadedTableNames []string) []
 			wg.Done()
 			continue
 		}
-		var loadTable bool
 		loadTable, err := job.shouldTableBeLoaded(tableName)
 		if err != nil {
-			panic(err)
+			loadErrors = append(loadErrors, err)
+			continue
 		}
 		if !loadTable {
 			wg.Done()
+			if misc.ContainsString(alwaysMarkExported, tableName) {
```

Comment:
instead of this, can we do something like `shouldTableBeLoaded` return reason and if reason is `NoLoadFiles` mark it as exported irresp of `alwaysMarkExported` list